components/Footer.vue

```diff
@@ -1,9 +1,59 @@
 <template>
 	<footer>
-		<ContactInfo />
-		<SocialMedia />
-		<Funders />
-		<p>
+		<div class="footer-content">
+			<nav aria-label="Footer" class="footer-nav">
+				<ul>
+					<li>
+						<nuxt-link to="/" aria-label="Go to Home">
+							Home
+						</nuxt-link>
```

Comment:
I don't think the `aria-label` is necessary. A link implicitly means "Go to [somewhere]". Also, overriding the link text with an `aria-label` means that a non-English user who is using Google Translate to translate the page will not get a translated version of the link text (`aria-label` doesn't get translated by translation tools: https://adrianroselli.com/2019/11/aria-label-does-not-translate.html).

Reply:
I agree, especially if it's clear that this is a pagination navigation section, the function of the number links is clear on its own.